src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -111,6 +111,13 @@ public Http2Stream(HttpRequestMessage request, Http2Connection connection, int s
                 {
                     // Create this here because it can be canceled before SendRequestBodyAsync is even called.
                     _requestBodyCancellationSource = new CancellationTokenSource();
+
+                    if (_request.HasHeaders && _request.Headers.ExpectContinue == true)
```

Comment:
Nit, as a matter of style we seem to avoid `== true` and similar

Reply:
This is a nullable bool; it's special.

Reply:
oh of course